### PR TITLE
Fix html tag in project-guidelines.md

### PR DESCRIPTION
**Pull request by alexperovich**
The <Reference> tag was getting parsed as an html tag and not showing in the view.

### CODE REVIEW

**Comment by danmosemsft** [comment]
You can do "skip ci please" for such things btw..

**Comment by danmosemsft** [comment]
Also you don't need to create a branch for markdown edits. Just edit directly and Github will give you the option of merging immediately into master, or it will make a branch for you and a PR for it.

**Comment by alexperovich** [comment]
I used the github ui to edit and had it create the PR. Do i put the "skip ci please" in the PR description?

File: Documentation/coding-guidelines/project-guidelines.md [hunk 1]
```diff
@@ -180,7 +180,7 @@ The output for the ref project build will be a flat targeting pack folder in the
 ##src
 In the src directory for a library there should be only **one** `.csproj` file that contains any information necessary to build the library in various configurations. All supported configurations should be listed in the `BuildConfigurations` property.
 
-All libraries should use '<Reference Include="..." />' for all their project references. That will cause them to be resolved against a targeting pack (i.e. `bin\ref\netcoreapp` or `\bin\ref\netstanard`) based on the project configuration. There should not be any direct project references to other libraries. The only exception to that rule right now is for partial facades which directly reference System.Private.CoreLib and thus need to directly reference other partial facades to avoid type conflicts.
```
Comment (weshaggard) [inline Documentation/coding-guidelines/project-guidelines.md]: I meant to use a backtick and not a single quote here so it gets formatted as code.

Comment (weshaggard) [inline Documentation/coding-guidelines/project-guidelines.md]: I switched it to backtick.

**Pull request merged**